atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/sweep/BackgroundSweeperImpl.java

```diff
@@ -332,8 +337,11 @@ private void saveSweepResults(final SweepProgressRowResult progress,
         if (cellsDeleted > 0) {
             Stopwatch watch = Stopwatch.createStarted();
             kvs.compactInternally(TableReference.createUnsafe(progress.getFullTableName()));
+            long elapsedMillis = watch.elapsed(TimeUnit.MILLISECONDS);
             log.debug("Finished performing compactInternally on {} in {} ms.",
-                    progress.getFullTableName(), watch.elapsed(TimeUnit.MILLISECONDS));
+                    progress.getFullTableName(), elapsedMillis);
+            sweepPerfLogger.logInternalCompaction(progress.getFullTableName(), cellsDeleted,
+                    cellsExamined, elapsedMillis);
```

Comment:
This could be a future improvement, but I'm not looking to refactor data structures here.